index.js

```diff
@@ -0,0 +1,67 @@
+import Tasks from './taskFunctions.js';
+import './style.css';
+import refresh from './refresh.svg';
+import enter from './enter-add.svg';
+
+const todoTitle = document.querySelector('.title');
+
+const addToList = document.querySelector('.add-task');
+
+const refreshIcon = document.createElement('img');
+const enterIcon = document.createElement('img');
+refreshIcon.src = refresh;
+enterIcon.src = enter;
+refreshIcon.className = 'refresh';
+enterIcon.className = 'enter';
+todoTitle.appendChild(refreshIcon);
+addToList.appendChild(enterIcon);
+
+const typeTask = document.getElementById('add-to-list');
+
+const todoList = new Tasks();
+
+window.addEventListener('load', () => {
+  todoList.retrieveTasks();
+});
+
+document.addEventListener('click', (event) => {
+  if (event.target === enterIcon && typeTask.value !== '') {
+    todoList.addTask(typeTask.value, false);
+  }
+  typeTask.value = '';
+});
+
+document.addEventListener('click', (event) => {
+  const deleteIcons = document.querySelectorAll('.menu');
+
+  deleteIcons.forEach((icon, index) => {
+    icon.addEventListener('click', () => {
+      todoList.removeTask(index);
+    });
+  });
+
+  if (event.target.matches('.clear-all')) {
+    todoList.clearCompleted();
+  }
+});

```

Comment:
As per the DRY principle, you could combine the two event listeners in one so as to avoid repeating the same code